examples/demo-advanced-steps.ipynb

```diff
@@ -17,57 +17,31 @@
     "## Setup"
```

Comment:
<p>This too got left</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Caltech-IPAC/firefly_client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>